Tests/Doctrine/Orm/OrderFilterTest.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+namespace Dunglas\ApiBundle\Tests\Doctrine\Orm;
+
+use Doctrine\ORM\EntityRepository;
+use Dunglas\ApiBundle\Api\Resource;
+use Dunglas\ApiBundle\Doctrine\Orm\OrderFilter;
+use Symfony\Bridge\Doctrine\Test\DoctrineTestHelper;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Test class for @see Dunglas\ApiBundle\Doctrine\Orm\OrderFilter.
```

Comment:
I actually found the proper annotation: it's `@coversDefaultClass` and `@covers`. I will change that tomorrow.